spec/tests/require.fmf

```diff
@@ -28,6 +28,10 @@ description: |
     ``destination`` to choose a different location. The ``nick``
     key can be used to override the default git repository name.
 
+    For debugging beakerlib libraries it is useful to reference
+    the development version directly from the local filesystem.
+    Use the ``path`` key to specify a full path to the library.
```

Comment:
Conflicting name with "path" from https://fmf.readthedocs.io/en/stable/concept.html#identifiers. Currently tmt doesn't support it( #446)

How about 'location' ?

Reply:
Ah, you are right - absolute vs relative.

Reply:
Good to merge then?